packages/integration-tests/local-registry/config.yml

```diff
@@ -8,12 +8,18 @@ auth:
 uplinks:
   npmjs:
     url: https://registry.npmjs.org/
-    max_fails: 20
+    max_fails: 30
     fail_timeout: 10m
+    timeout: 60s
+    cache: false
+    maxage: 30m
   yarn:
     url: https://registry.yarnpkg.com
-    max_fails: 20
+    max_fails: 30
     fail_timeout: 10m
+    timeout: 60s
+    cache: false
+    maxage: 30m
```

Comment:
You missed the `keep-alive` is the one which actually boots performance.

Reply:
https://github.com/verdaccio/verdaccio/blob/master/conf/default.yaml#L39

Reply:
Also this
```
logs:
  - { type: stdout, format: pretty, level: http }
  ```
slow down a lot, don't use `logs` on CI, change `http` to `warn`.

Reply:
Sorry maybe I misunderstood how the config gets applied then, I had assumed default config meant I didn't have to enable it? Are you saying that by specifying any kind of uplink in my own config I am no longer getting any of the default values?

Reply:
Yes, currently the keep alive is not built in, you must defined it. In the next minor would be built in then you don't need that configuration anymore (only keep alive)

Reply:
Ah ok thanks

Reply:
And thanks I've updated the logs config